extension/src/repository/index.ts

```diff
@@ -45,7 +48,10 @@ export class Repository extends DeferredDisposable {
   }
 
   public getState() {
```

Comment:
I'm going to follow up and remove this. It is only used for testing purposes. This change is big enough now though.